README.md

```diff
@@ -35,7 +35,13 @@ The JAXsim API documentation is available at [jaxsim.readthedocs.io](https://jax
 
 ## Installation
 
-You can install the project with [`pypa/pip`][pip], preferably in a [virtual environment][venv]:
+You can install the project using [`conda`][conda]:
+
+```bash
+conda install jaxsim -c conda-forge 
```

Comment:
```suggestion
conda install jaxsim -c conda-forge
```

Reply:
@flferretti can you setup you editor to remove automatically blank trailing spaces?